.github/workflows/generator.yaml

```diff
@@ -0,0 +1,81 @@
+name: Client Generator
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        go-version: [ '1.23' ]
+    defaults:
+      run:
+        working-directory: generator
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup Go ${{ matrix.go-version }}
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ matrix.go-version }}
+        cache-dependency-path: |
+          generator/go.sum
+    - name: Display Go version
+      run: go version
+    - run: go mod download
```

Comment:
use `go mod tidy` instead (in general I almost never run` go mod download`)

Reply:
Done.